services/softlayer_network_storage.go

```diff
@@ -45,10 +45,10 @@ func (slns *softLayer_Network_Storage_Service) CreateIscsiVolume(size int, locat
 
 	iopsItemPriceId := slns.getPerformanceStorageItemPriceIdByIops(size)
 
-	order := datatypes.SoftLayer_Product_Order{
+	order := datatypes.SoftLayer_Container_Product_Order_Network_PerformanceStorage_Iscsi{
 		Location:    location,
 		ComplexType: "SoftLayer_Container_Product_Order_Network_PerformanceStorage_Iscsi",
-		OsFormatType: datatypes.OsFormatType{
+		OsFormatType: datatypes.SoftLayer_Network_Storage_Iscsi_OS_Type{
 			Id:      12,
```

Comment:
@maximilien 
Here is a sample instance for `SoftLayer_Network_Storage_Iscsi_OS_Type `

`osType":{"createDate":"2014-07-16T21:22:25+08:00","description":"Use if your host operating system is Linux.","id":12,"keyName":"LINUX","name":"Linux"}`

The `Id` for `Linux` os type is `12`, that is really what we want during ordering the iscsi storage, and it is constant, no longer change in future, so I just hardcoded it here.